docs/src/enterprise/getstarted/install.md

```diff
@@ -278,92 +250,69 @@ replace with your IdP details.
             user_filter: (objectClass=inetOrgPerson)
             connection_timeout_seconds: 15
             request_timeout_seconds: 17
-      secrets:
-        create: true
-      sso:
-        enabled: true
-        ldap:
-          enabled: true
-          bind_password: <ldap bind password>
-      rbac:
-        enabled: true
+            # RBAC group for first time users
+            default_user_group: "Developers"
+
+    ingress:
+      enabled: true
+      ingressClassName: <class-name>
+      hosts:
+        - host: <lakefs.acme.com>
+          paths:
+            - /
 
-    useDevPostgres: true
     ```
 
-See [additional examples on GitHub](https://github.com/treeverse/charts/tree/master/examples/lakefs/enterprise) we provide for each authentication method (oidc, adfs, ldap, rbac, IAM etc).
+See [additional examples on GitHub](https://github.com/treeverse/charts/tree/master/examples/lakefs/enterprise) we provide for each authentication method (oidc, saml, ldap, rbac, external AWS IAM).
 
 ### Database Configuration
 
-In this section, you will learn how to configure lakeFS Enterprise to work with the KV Database you created (see [prerequisites](#prerequisites).
+In this section, you will learn how to configure lakeFS Enterprise to work with the KV Database you created (see [prerequisites](#prerequisites)).
 
 Notes:
 
-* By default, the lakeFS Helm chart comes with `useDevPostgres: true`, you should change it to `useDevPostgres: false` for Fluffy to work with your KV Database and be suitable for production needs.
-* The KV database is shared between lakeFS and Fluffy, and therefore both services must use the same configuration.
-* See [fluffy][fluffy-configuration] and [lakeFS](../../reference/configuration.md#database) `database` configuration.
+* By default, the lakeFS Helm chart comes with `useDevPostgres: false`, you can change it to `useDevPostgres: true` for dev use.
```

Comment:
Explain why would anyone set it to true (multiple replicas), prevent data loss between container restarts